Light Vox Engine/Engine.cpp

```diff
@@ -14,16 +14,17 @@
 Engine* Engine::engineInstance = nullptr;
 
 /* LIFE CYCLE */
-Engine::Engine(LV_INSTANCE hInstance )
+Engine::Engine( LV_INSTANCE hInstance )
 {
-	if (hInstance != nullptr) {
-		this->hInstance = hInstance;
-		Engine::engineInstance = this;
-		windowWidth = 1280;
-		windowHeight = 720;
-		windowTitle = "STRUGGLE BUS";
-		hWindow = 0;
-	}
+    if ( hInstance != nullptr )
```

Comment:
hInstance is Windows only

Reply:
it's an LV_INSTANCE so it'll be typedef'ed to the appropriate thing on Windows and differently on other platforms